### ISSUE TITLE
ReShade causes games to freeze if I hold down a key on the keyboard and move the mouse at the same time

### ISSUE THREAD

**Issue by DoomPenguin9** [kevinlekiller/reshade-steam-proton#17]
Whis issue happens in every single game I've used ReShade in. It happens under both regular Wine and Proton. Games I've tested:

Elden Ring
Dark Souls 3
S.T.A.L.K.E.R: Anomaly
S.T.A.L.K.E.R: Radiophobia 3
Battlefield 4

This is not an issue fo DS3 and ER because those are games which I play on a controller and this issue does not occur if I use a controller, but playing FPS games with ReShade is impossible since I need KB+Mouse input for those.

To reproduce this issue, just star a game with some ReShade effects applied, hold the W key, and look around with the mouse. When I do this, the games would freeze for a solid 30-40 seconds.

**Comment by timoleistner** [comment]
I'm having the same problem on an AMD GPU (AMD ATI Radeon RX Vega 56/64) if this is relevant.

**Comment by timoleistner** [comment]
Okay I've finally found a solution.
By installing different mods for S.T.A.L.K.E.R: Anomaly I've noticed that this seems to be a [ReShade](https://github.com/kevinlekiller/reshade-steam-proton/issues/17) problem.
Using a custom wine build downloaded from [here](https://github.com/Kron4ek/Wine-Builds).
**Use a wine version lower than 5.14 and the freezes are gone.**
The setting in Lutris for a custom wine binary are a bit hidden behind the "advanced options" box.
